### PR TITLE
feat: Make lite version

### PR DESCRIPTION
**Pull request by internetova**
## Changes
- removed Suggestions, Search, ReverseSearch, Bicycle, Driving
- updated native files
- updated example
- updated others files

## Video

https://github.com/surfstudio/yandex-mapkit-lite-flutter/assets/39735343/8dc5a6ab-38a5-42e5-baf6-dc12222635be

## Checklist for self-check
- [x] Commits and PRs have been filed according to [the rules on the project](https://github.com/surfstudio/surf-flutter-app-template#workflow-in-a-repository).
- [x] The author is marked as an assigne and assigned mandatory reviewers.
- [x] Required labels marked
- [x] Specified related tasks and/or related PRs.
- [x] Specified Changes.
- [x] Attached videos/screenshots demonstrating the fix/feature.
- [x] All unspecified fields in the PR description deleted.
- [ ] New code covered by tests.

## Checklist for reviewers
- [ ] CI passed successfully _(with a green check mark)_.
- [ ] PR is atomic, by volume no more than 400 (+-) corrected lines (not including codogen).

Design:
- [ ] System design corresponds to the agreements on structure and architecture on the project.
- [ ] The code is decomposed into necessary and sufficient components.

Functionality:
- [ ] The code solves the problem.
- [ ] Any changes to the user interface are reasonable and look good.

Complexity:
- [ ] The code is clear, easy to read, functions are small, no more than 50 lines.
- [ ] The logic is not overcomplicated, there is no overengineering (no code sections that may be needed in the future, but no one knows about it).

Tests:
- [ ] Updated or added tests for mandatory components.
- [ ] The tests are correct, helpful, and well designed/developed.

Naming:
- [ ] The naming of variables, methods, classes and other components is understandable.

Comments:
- [ ] The comments are understandable and helpful.

Documentation:
- [ ] All labels are correct
- [ ] Technical documentation updated (after approval, updates last reviewer).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# yandex_mapkit
+# yandex_mapkit_lite
+
+**This is a simplified version of the yandex_mapkit 3.4.0 package.**
+Features such as Suggestions, Search, ReverseSearch, Bicycle, Driving have been removed from it.
```
Comment (maxim-sysoev) [inline README.md:4]: It is necessary to explicitly mark that this library is a fork.
It is necessary to describe in more detail the difference from the full version of maps (what functions are missing, if they are needed, we recommend using mapkit full), in what cases it should be used. And provide a link to a specific mapkit package.

File: README.md [hunk 2]
```diff
@@ -138,8 +141,5 @@ AndroidYandexMap.useAndroidViewSurface = false;
 * [X] Setting map bounds
 * [X] Showing current user location
 * [X] Styling the map
-* [X] Address suggestions
-* [X] Basic driving/bicycle routing
-* [X] Basic address direct/reverse search
 * [X] Workking with geo objects
-* [X] Showing current traffic conditions
+*
```
Comment (maxim-sysoev) [inline README.md:145]: empty point

File: README.md (hunk 1 above)
Comment (internetova) [reply README.md:27]: ✅

File: README.md (hunk 2 above)
Comment (internetova) [reply README.md]: ✅

**Review by Sadhorsephile — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,4 +1,30 @@
-# yandex_mapkit
+# yandex_mapkit_lite
+
+**This library is a fork of the package yandex_mapkit 3.4.0.**
+Features such as Suggestions, Search, ReverseSearch, Bicycle, Driving have been removed from it.
```
[the remark this answers is not in the document]
Comment (internetova) [reply README.md:4]: ✅

**Review by maxim-sysoev — APPROVED** [review]

**Pull request merged**